src/components/PasswordSetterBox.js

```diff
@@ -148,15 +177,14 @@ class PasswordSetterBox extends Nimiq.Observable {
     }
 
     _isEligiblePassword() {
-        const password = this._passwordInput.text;
-        if (password.length < PasswordInput.DEFAULT_MIN_LENGTH) return false;
-        return PasswordStrength.strength(password) >= PasswordStrength.Score.MINIMUM;
+        if (this._passwordInput.text.length < PasswordInput.DEFAULT_MIN_LENGTH) return false;
+        return true;
     }
 
     /**
      * @param {Event} event
      */
-    _onSubmit(event) {
+    async _onSubmit(event) {
         event.preventDefault();
```

Comment:
Maybe this function should also not be executed during password repeat (just like the button is hidden, ENTER should then also not work). So there could be `if (this._password) return` here.

I'm not sure it should be though, so this is not a change request. More a discussion starter.